### ISSUE TITLE
Bug macos Signet setup Recory path multisigner in new wallet

### ISSUE THREAD

**Issue by GaloisField2718** [wizardsardine/liana#928]
Hello,

**Config** : 
- MacOS X 14.0
- Intel i5

Following the presentation of LIANA by @pythcoiner , I got this issue : 

```
Command to start bitcoind returned an error : Cookie file was not found at the expected path : /Users/galoisfield/Library/Application Support/Liana/bitcoind/datadir/signet/.cookie
```

I tried to launch from terminal (`/Users/galoisfield/Library/Application Support/Liana/bitcoind`):
 `./bin/bitcoind -conf=datadir/bitcoin.conf`

More or less the same error each time : 
```
Error: Error reading configuration file: specified config file "/Users/galoisfield/Library/Application Support/Bitcoin/datadir/bitcoin.conf" could not be opened
```

Let me know if you need more info.

**Comment by pythcoiner** [comment]
can you try w/ absolute path in `-conf` arg?

**Comment by GaloisField2718** [comment]
Without any argument I reach : `Error: Error reading configuration file: conf cannot be set in the configuration file; use includeconf= if you want to include additional config files`

When I specify the absolute path : I get an error from `datadir`
```
Error: Data directory "/Users/galoisfield/Library/Application Support/Bitcoin" contains a "bitcoin.conf" file which is ignored, because a different configuration file "/Users/galoisfield/Library/Application Support/Liana/bitcoind/datadir/bitcoin.conf" from command line argument "-conf=/Users/galoisfield/Library/Application Support/Liana/bitcoind/datadir/bitcoin.conf" is being used instead. Possible ways to address this would be to:
- Delete or rename the "bitcoin.conf" file in data directory "/Users/galoisfield/Library/Application Support/Bitcoin".
- Change datadir= or conf= options to specify one configuration file, not two, and use includeconf= to include any other configuration files.
- Set allowignoredconf=1 option to treat this condition as a warning, not an error.
```

**Comment by jp1ac4** [comment]
> Following the presentation of LIANA by @pythcoiner , I got this issue :
> 
> ```
> Command to start bitcoind returned an error : Cookie file was not found at the expected path : /Users/galoisfield/Library/Application Support/Liana/bitcoind/datadir/signet/.cookie
> ```

Was this error received after choosing the Liana-managed Bitcoin node option when creating your wallet?

Regarding running it from the terminal, I think you might need to pass `-datadir` instead of `-conf`.

**Comment by GaloisField2718** [comment]
> > Following the presentation of LIANA by @pythcoiner , I got this issue :
> 
> > 
> 
> > ```
> 
> > Command to start bitcoind returned an error : Cookie file was not found at the expected path : /Users/galoisfield/Library/Application Support/Liana/bitcoind/datadir/signet/.cookie
> 
> > ```
> 
> 
> 
> Was this error received after choosing the Liana-managed Bitcoin node option when creating your wallet?
> 
> 
> 
> Regarding running it from the terminal, I think you might need to pass `-datadir` instead of `-conf`.

Ah ! Instead of `-conf` flag ?!

**Comment by pythcoiner** [comment]
wich version of MacOs are you running? i'll try to launch a VM to try reproduce

**Comment by pythcoiner** [comment]
maybe related to https://github.com/wizardsardine/liana/issues/929#issuecomment-1900021680, will try to reproduce in a VM.

@GaloisField2718, can you check wich bitcoind version you run ?
you can check the content of `/Users/galoisfield/Library/Application Support/Liana/bitcoind/`, oerhaps you have several versions?

**Comment by pythcoiner** [comment]
I reach to reproduce on linux, the steps are:
- Make a new signet wallet
- Make full IBD
- Stop Liana
- Manually remove .liana/signet
- Manually remove .liana/bitcoind/datadir/signet

Then when try to setup a new signet wallet we have this error:

![image](https://github.com/wizardsardine/liana/assets/124568858/8f1fdb0c-8da2-4380-95e6-b2eecbe7e1ab)

**Comment by pythcoiner** [comment]
@GaloisField2718 did you come to this issue by this way?

**Comment by GaloisField2718** [comment]
> @GaloisField2718 did you come to this issue by this way?

I doesn't specifically removed .liana folders but I opened liana, launched signet, close liana and open again to use another network with specific wallet recovery settings.

**Comment by pythcoiner** [comment]
It looks to me it was a bitcoin core auth  w/ cookie [issue](https://github.com/bitcoin/bitcoin/pull/28784) that have been fixed in 26.1.
So i close it, anyone feel free to reopen if upgrading to bitcoind 26.1 don't fix on your side.
